app/views/catalog/_index_list_default.html.erb

```diff
@@ -0,0 +1,19 @@
+    <div class="col-sm-6">
+      <table class="table attribute-table">
+        <% doc_presenter = index_presenter(document) %>
+        <% index_fields(document).each do |field_name, field| -%>
+          <% if should_render_index_field? document, field %>
+            <tr>
+              <th><span class="attribute-label h4"><%= render_index_field_label document, field: field_name %></span></th>
+              <td><%= doc_presenter.field_value field_name %></td>
```

Comment:
Can we update the span class so that the table > th font-size is smaller than the h3 document-title-heading class?

Reply:
:+1: Done.

Reply:
do we need the h4 in the class? if you remove it the size is much better. identifier in the screenshot has no h4
![Screen Shot 2019-03-19 at 1 24 28 PM](https://user-images.githubusercontent.com/9905193/54627787-80c5e900-4a4a-11e9-906e-7452642a8377.png)

Reply:
Sure, I can do that. Done.